test/integration-test/CodeGen/coriolis_stencil_benchmark.cpp

```diff
@@ -36,6 +36,7 @@ TEST(coriolis_stencil, test) {
   verif.fillMath(5.0, 1.2, 1.3, 1.7, 2.2, 3.5, v_nnow);
 
   gridtools::coriolis_stencil coriolis_gt(dom, u_tens_gt, u_nnow, v_tens_gt, v_nnow, fc);
+  verif.sync_storages(u_nnow, v_nnow, fc);
```

Comment:
this is between the initializations, not the runs

Reply:
yes, done